pkg/services/validationService.go

```diff
@@ -97,11 +102,22 @@ func Validate(admissionReviewReq *admission.AdmissionReview) *admission.Admissio
 	evaluationRequestData := getEvaluationRequestData(token, clientId, clusterK8sVersion, policy.Name, startTime,
 		policyCheckResults)
 
+	verifyVersionResponse, err := cliClient.VerifyWebhookVersion(evaluationRequestData.WebhookVersion)
+	if err != nil {
+		warningMessages = append(warningMessages, err.Error())
+	} else {
+		if verifyVersionResponse.MessageTextArray != nil {
```

Comment:
double check me, but if I'm not mistaken golang's default empty value for arrays is an empty array and not a `nil` value (not like `undefined` in js)
so this check can be redundant if `MessageTextArray` is not a pointer (I think)